conftest.py

```diff
@@ -10,17 +10,48 @@
 import pytest
 import pytest_asyncio
 
-from hatchet_sdk import Hatchet
+from hatchet_sdk import ClientConfig, Hatchet
+from hatchet_sdk.loader import ClientTLSConfig
+
+
+@pytest.fixture(scope="session")
+def token() -> str:
+    result = subprocess.run(
+        [
+            "docker",
+            "compose",
+            "run",
+            "--no-deps",
+            "setup-config",
+            "/hatchet/hatchet-admin",
+            "token",
+            "create",
+            "--config",
+            "/hatchet/config",
+            "--tenant-id",
+            "707d0855-80ab-4e1f-a156-f1c4546cbf52",
+        ],
+        capture_output=True,
+        text=True,
+    )
+
+    return result.stdout.strip()
```

Comment:
FYI @grutt - this should be a fix to hard-coding that token env var (the thing we talked about a week or two ago - [this](https://github.com/hatchet-dev/hatchet-python-v2/pull/8/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L64-L66))